misc/acrn-config/library/scenario_cfg_lib.py

```diff
@@ -809,3 +809,56 @@ def check_p2sb(enable_p2sb):
         if p2sb and board_cfg_lib.is_tpm_passthru():
             ERR_LIST["vm:id=0,p2sb"] = "Cannot enable p2sb and tpm passthru at the same time"
             return
+
+
+def check_pt_intx(phys_pin, virt_pin):
+
+    if not phys_pin and not virt_pin:
+        return
+
+    (_, board) = common.get_board_name()
+    if board != "ehl-crb-b":
+        ERR_LIST["pt_intx"] = "only board ehl-crb-b is supported"
+        return
+
+    (_, scenario) = common.get_scenario_name()
+    if scenario != "hybrid" and scenario != "hybrid_rt":
```

Comment:
can't work on a pre-launched VM0 of other scenarios?

Reply:
per subject "expose GPIO Chassis Interrupt to the Safety VM as INTx", if this is for safety VM only, then we need to add condition that the VM0 is a safety VM, and "hybrid_rt" does not satisfy this condition.